bindings/gumjs/gumscript-runtime-objc.js

```diff
@@ -1031,6 +1089,9 @@
                 if (specifier.hasOwnProperty('subclasses'))
                     subclasses = specifier.subclasses;
             }
+            if (typeof cls === 'string') {
```

Comment:
I see, but it is a lot longer to write `ObjC.classes.NSString` instead of just `'NSString'`, so I think we should keep it for ease of use.